### PR TITLE
Simplify example: Box plots with custom fill colors

### PR DESCRIPTION
**Pull request by timhoffm**
The [original example](https://matplotlib.org/3.8.0/gallery/statistics/boxplot_color.html) was more complex than need be:

- The two plots only differed in the `notch=True` parameter, which is irrelevant for custom fill colors -> reduce to one
- rework to a more realistic-like plot

### CODE REVIEW

File: galleries/examples/statistics/boxplot_color.py [hunk 1]
```diff
@@ -3,52 +3,34 @@
...
-                     vert=True,  # vertical box alignment
-                     patch_artist=True,  # fill with color
-                     labels=labels)  # will be used to label x-ticks
-ax2.set_title('Notched box plot')
+fruit_weights = [
+    np.random.normal(130, 10, size=100),
+    np.random.normal(125, 20, size=100),
+    np.random.normal(120, 30, size=100),
+]
+labels = ['peaches', 'oranges', 'tomatos']
```
Comment (rcomer) [inline galleries/examples/statistics/boxplot_color.py:21]:
```suggestion
labels = ['peaches', 'oranges', 'tomatoes']
```

File: galleries/examples/statistics/boxplot_color.py [hunk 2]
```diff
@@ -3,52 +3,34 @@
...
+colors = ['peachpuff', 'orange', 'tomato']
```
Comment (rcomer) [inline galleries/examples/statistics/boxplot_color.py:22]: I did not know these colours existed by using them here is 😎

**Review by rcomer — APPROVED** [review]
One small typo but otherwise looks good to me.

File: galleries/examples/statistics/boxplot_color.py (hunk 2 above)
Comment (timhoffm) [reply galleries/examples/statistics/boxplot_color.py:22]: Actually, I checked https://matplotlib.org/devdocs/gallery/color/named_colors.html and chose the fruits after available color names 😎

**Pull request merged**